src/rubrix/client/models.py

```diff
@@ -89,7 +89,7 @@ class TextClassificationRecord(BaseModel):
             The timestamp of the record.
     """
 
-    inputs: Dict[str, Union[str, List[str]]]
+    inputs: Union[Union[str, List[str], Dict[str, Union[str, List[str]]]]]
```

Comment:
I think there is one `Union` too many
```suggestion
    inputs: Union[str, List[str], Dict[str, Union[str, List[str]]]]
```